### PR TITLE
Made JS standard-compliant, standard added as dev-dependency

### LINKED ISSUE #812: Make JavaScript comply with Standard linting
Per my discussion with @ssciolla here: https://github.com/tl-its-umich-edu/my-learning-analytics/pull/775, I'm going to make every file conform to the linting rules of [Standard](https://standardjs.com/) and add a note about linting in the `CONTRIBUTING` doc.

### PR DESCRIPTION
**Pull request by justin0022**
Resolves #812 

There's many file changes in this PR, but there _shouldn't_ be any changes to the underlying JS semantics. I did some basic testing and didn't notice any issues.

### CODE REVIEW

File: assets/src/containers/ResourcesAccessed.js [hunk 1]
```diff
@@ -250,23 +250,22 @@ function ResourcesAccessed (props) {
...
-                    checked={saveSettingClicked}
-                    onChange={() => setSaveSettingClicked(!saveSettingClicked)}
-                    value='checked'
-                    color='primary'/>
-              : <div style={{ padding: '10px' }}></div>
+                ? <Checkbox
+                  checked={saveSettingClicked}
+                  onChange={() => setSaveSettingClicked(!saveSettingClicked)}
+                  value='checked'
+                  color='primary' />
```
Comment (ssciolla) [inline assets/src/containers/ResourcesAccessed.js]: This seems to be a weird case where `standard` wants the `/>` to be both aligned with the tag beginning and to be un-indented once before it. Was that your experience? Is this a known problem with `standard`?

**Comment by justin0022** [comment]
Thanks I’ll resolve and merge on Monday

File: assets/src/containers/ResourcesAccessed.js (hunk 1 above)
Comment (justin0022) [reply assets/src/containers/ResourcesAccessed.js]: For the `/>` I've been putting it on a new line if the component is multi-line. For example:
```js
<Checkbox
  checked={saveSettingClicked}
  onChange={() => setSaveSettingClicked(!saveSettingClicked)}
  value='checked'
  color='secondary'
/>
```
If the component is one-line, then the closing tag goes at the end with a space.
```js
<div style={{ padding: '10px' }} />
```

**Comment by justin0022** [comment]
@ssciolla it should be ready to merge now, please review when you can!

**Comment by justin0022** [comment]
Good catch, I'll fix it right now

**Pull request merged**